src/microsoft-dotnet-interactive/src/javascriptKernel.ts

```diff
@@ -74,10 +74,18 @@ export class JavascriptKernel extends kernel.Kernel {
 
     private allLocalVariableNames(): string[] {
         const result: string[] = [];
-        for (const key in globalThis) {
-            if (typeof (<any>globalThis)[key] !== 'function') {
-                result.push(key);
+        try {
```

Comment:
@brettfo I added this because we are getting dom security exceptions during this scanning and they break the constructor.

This is the kind of error we get in headless js kernel 
https://github.com/microsoft/playwright/issues/9934